quickshop-bukkit/src/main/java/com/ghostchu/quickshop/shop/display/virtual/VirtualDisplayItem.java

```diff
@@ -106,30 +105,7 @@ public ItemStack checkEnchants(final ItemStack itemStack) {
       cloned.getEnchantments().clear();
       return cloned;
     }
-
-    final int amount = cloned.getEnchantments().size();
-    boolean removed = false;
-
-    for(final Map.Entry<Enchantment, Integer> entry : itemStack.getEnchantments().entrySet()) {
-
-      final Enchantment enchantment = entry.getKey();
-
-      if(EnchantmentTypes.getRegistry().getByName(enchantment.key().asString()) == null) {
-        cloned.removeEnchantment(enchantment);
-        removed = true;
-        continue;
-      }
-      if(EnchantmentTypes.getByName(enchantment.key().asString()) == null) {
-        cloned.removeEnchantment(enchantment);
-        removed = true;
-      }
-    }
-
-    if(amount == 1 && removed) {
-      cloned.addUnsafeEnchantment(Enchantment.UNBREAKING, 1);
-    }
-
-    return cloned;
+    return manager.packetHandler() != null ? manager.packetHandler().filterEnchantments(cloned) : cloned;
```

Comment:
"manager.packetHandler() != null ?" should be enclosed according to the QS Style guidelines with no space before the question mark portion of the ternary operator

(manager.packetHandler() != null)?